README.md

```diff
@@ -17,15 +17,22 @@ query whether or not their versions are impacted.
   <img src="docs/images/diagram.png" width="600">
 </p>
 
-Current data sources:
-- [OSS-Fuzz](https://github.com/google/oss-fuzz) (https://github.com/google/oss-fuzz-vulns)
+## Current data sources
+**This is an ongoing project.** We are hoping to work with the open source community
```

Comment:
Can use slightly stronger tone. like "We encourage open source ecosystems to try our proposed vulnerability interchange schema format (link) and integrate with our existing database, for the benefit of the open source community. The following ecosystems have accepted our schema format and integrated with us:"

Reply:
Done. Also linked to blog post. PTAL!